src/webui/webui.cpp

```diff
@@ -105,10 +103,9 @@ void WebUI::init()
                 const QString errorMsg = tr("Web UI: Unable to bind to IP: %1, port: %2. Reason: %3")
                     .arg(serverAddressString).arg(m_port).arg(m_httpServer->errorString());
                 logger->addMessage(errorMsg, Log::CRITICAL);
-#ifdef DISABLE_GUI
-                qCritical() << errorMsg;
```

Comment:
this is most important part! please add back to where you see fit.

Reply:
Oops.. It was shrinked with `QCoreApplication::exit(1);`.

Reply:
Fixed.